.github/workflows/stylelint.yml

```
@@ -0,0 +1,49 @@
+name: CSS Coding Standards
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  stylelint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: "npm"
+
+      - name: Get npm cache directory
+        id: npm-cache-dir
+        run: echo "dir=$(npm config get cache)" >> ${GITHUB_OUTPUT}
+
+      - name: Cache npm dependencies
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.npm-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Check Node version
+        run: node -v
+
+      - name: Setup NPM
+        run: npm install -g npm@latest
+
+      - name: Check NPM version
+        run: npm -v
+
+      - name: Install dependencies
+        run: npm install
```

```suggestion
        run: npm ci
```

I'd run CI here to make sure it installs the exact version from the lock file

---

done